packages/itwinui-react/jest.config.js

```diff
@@ -83,7 +83,7 @@ module.exports = {
 
   // A map from regular expressions to module names or to arrays of module names that allow to stub out resources with a single module
   moduleNameMapper: {
-    '\\.(css|less|sass|scss)$': '<rootDir>/src/__mocks__/styleMock.js',
+    'styles.js$': '<rootDir>/src/__mocks__/styleMock.js',
```

Comment:
This changes the way the css is imported into React right? Through `styles.js` file instead of the files in the old folder?

Reply:
no, this file is only used for configuring `jest` unit tests.

specifically this line is used to return a dummy `styleMock.js` instead of the real `styles.js` file (which doesn't exist in source code).

you can read about this option here: https://jestjs.io/docs/configuration#modulenamemapper-objectstring-string--arraystring